db/schema.rb

```diff
@@ -36,7 +36,7 @@
 
   create_table "invoices", force: :cascade do |t|
     t.bigint "customer_id"
-    t.integer "status", default: 3
+    t.integer "status", default: 0
```

Comment:
We've been using the default of 3 for a bunch of tests, did this get updated everywhere?